lib/facter/rh_certificates.rb

```diff
@@ -4,7 +4,7 @@ def extract_certificates_from_subscription_manager
   private_key_end_path = '/key.pem'
   rh_default_ca_cert = '/etc/rhsm/ca/redhat-uep.pem'
   data = Facter::Util::Resolution.exec('subscription-manager config')
-  return nil if data.nil?
+  return nil if data.nil? || data.blank?
```

Comment:
I am so sorry for the confusion I made on #20 ... `blank?` is a Rails specific method. Can you please change this to `.empty?` which is a ruby method? Thanks and sorry